cardano-sl.cabal

```diff
@@ -30,6 +30,12 @@ Flag with-explorer
 
   description: Build with explorer
 
+flag bench
```

Comment:
I pushed a branch without this flag, the build hasn't started yet but will start sooner or later.
It's here https://travis-ci.org/input-output-hk/cardano-sl/builds/261750906